client/rpc/contract.go

```diff
@@ -10,7 +10,8 @@ import (
 
 // ContractApi ...
 type ContractApi interface {
-	CallOffChainMethod(param api.CallOffChainMethodParam) ([]byte, error)
+	CallOffChainMethod(param api.CallOffChainMethodParam) ([]byte, error)  // Deprecated: Use Query() instead
```

Comment:
https://github.com/vitelabs/go-vite/blob/master/client/rpc/contract.go
This feat has been merged to the master. Why is it still showing up here?

Reply:
They are not exactly the same commit, the function `vm.NewVM` is refactored by adding an additional parameter (from `vm.NewVM(nil)` to `vm.NewVM(nil, nil)`)
The previous commit is a backward-compatible version after resolving conflicts.